1-exercises/A-accessing-values/exercise3.js

```diff
@@ -19,7 +19,8 @@ let basketballTeam = {
     - sorts the top players in alphabetical order
     - console.logs the name of each player on a new line
 */
-
+let array=basketballTeam.topPlayers;
+console.log(array.sort());
 // write code here
```

Comment:
Great job, we wanted to see the name of each player so you should do this instead: 

`basketballTeam.topPlayers.sort().map(eachPlayer =>. console.log(eachPlayer))`

Reply:
Its concise and so neat ! I will try and refactor mine